Frontend/app/secure-test/secure-test.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component } from '@angular/core';
```

Comment:
I think you may have put this comment on the wrong line but I assume youre talking about "secure-test" and yeah it doesn't follow the convention. I can change it or leave it based on what we decide but this class will ideally be deleted by the end of the next sprint since it is just used for testing and as an example of how to use the new AuthHttp service.